src/main/resources/db/migration/V1__create_tables.sql

```diff
@@ -0,0 +1,77 @@
+CREATE TYPE PHONE_TYPE as ENUM ('home', 'work', 'mobile');
+
+CREATE TYPE ROLE as ENUM ('associate', 'partner', 'senior');
+
+CREATE TYPE STATUS as ENUM ('new', 'assigned', 'on_hold',
+    'approved', 'canceled', 'declined');
+
+CREATE TABLE users
+(
+    id               bigserial
+        constraint PK_users PRIMARY KEY,
+    email            text not null
+        constraint UQ_users_email UNIQUE ,
+    cognito_username text not null
+        constraint UQ_users_cognito_username UNIQUE
+);
+
+CREATE TABLE advisors
+(
+    id   bigserial
```

Comment:
BIGSERIAL is not a true data type but rather a convenience notation that tells PostgreSQL to create an auto-incrementing integer column. As you want to use shared key, change to BIGINT